webhook_server_container/libs/github_api.py

```diff
@@ -2056,20 +2058,18 @@ def owners_data_for_changed_files(self) -> dict[str, dict[str, Any]]:
         data: dict[str, dict[str, Any]] = {}
 
         changed_folders = {Path(cf).parent for cf in self.changed_files}
-
+        all_owner_data = self.all_approvers_and_reviewers
```

Comment:
why is this assignment needed?

Reply:
I need to change these based on what you suggested. This PR is not ready for review. I will add a wip.